agent-sdk/src/main/java/co/elastic/otel/android/internal/features/exportergate/ExporterGateManager.kt

```diff
@@ -150,8 +150,8 @@ internal class ExporterGateManager(
                 delegateSpanExporter.export(items)
             }
             gateSpanExporter = null
+            spanGateOpen.set(true)
```

Comment:
Is this related to the release process in the CI?

Reply:
No, the changes in this file were added later when I saw some failed GH checks, and I think it's due to some flaky tests that should get sorted by these changes.